jdaviz/app.py

```diff
@@ -887,6 +848,179 @@ def _get_all_subregions(mask, spec_axis_data):
 
         return regions
 
+    def get_subsets(self, subset_name=None, spectral_only=False,
```

Comment:
will there eventually be methods at the helper-level as well, or is this intended to stay at the app-level?

Reply:
Yes, there is already `specviz.get_spectral_regions()` that uses this. My intention is to use this in the Subset Plugin as well.